pypeit/spectrographs/lbt_mods.py

```diff
@@ -431,7 +431,7 @@ def get_detector_par(self, hdu, det):
             det=1,
             dataext         = 0,
             specaxis        = 0,
-            specflip        = False,
+            specflip        = True,
```

Comment:
How was this ever working before if the wavelength axis was flipped. I don't see how you could have gotten a wavelength solution out of full_template with flipped wavelength direction.

Reply:
Ah okay, now I see you checked in a new arxiv, so I guess you flipped it.